executor/local/build.go

```diff
@@ -21,171 +21,131 @@ import (
 
 // CreateBuild configures the build for execution.
 func (c *client) CreateBuild(ctx context.Context) error {
-	b := c.build
-	p := c.pipeline
-	r := c.repo
-	e := c.err
-
 	// defer taking snapshot of build
-	defer build.Snapshot(b, nil, e, nil, r)
+	defer build.Snapshot(c.build, nil, c.err, nil, nil)
```

Comment:
are `c.err` and `c.repo` no longer needed in the `Snapshot()`?

Reply:
https://github.com/go-vela/pkg-executor/blob/7b26d6a726dd026d465ae4ce8c072bcece794a4b/internal/build/snapshot.go#L39

Reply:
i think it's fine since client is `nil`, it only uses repo if client is passed in, afaict

Reply:
oh duh! client has always been nil, didnt catch that, got it 👍